.github/workflows/pr-comment.yaml

```diff
@@ -48,9 +48,7 @@ jobs:
         id: generate-comment
         if: steps.download-ecosystem-result.outputs.found_artifact == 'true'
         run: |
-          echo '## PR Check Results' >> comment.txt
-
-          echo "### Ecosystem" >> comment.txt
+          echo '## `ruff-ecosystem` results' >> comment.txt
           cat pr/ecosystem/ecosystem-result >> comment.txt
```

Comment:
These changes will not take effect until after merge. I can manually dispatch to test if desired.

Reply:
Doing a manual dispatch might be good. Just be to sure